src/partials/head-meta.hbs

```diff
@@ -1 +1,16 @@
     {{!-- Add additional meta tags here --}}
+    <meta property="og:site_name" content="DataStax Docs">
+    <meta property="og:type" content="website">
+    <meta property="og:url" content="{{site.url}}">
+    <meta property="og:title" content="{{page.title}} :: {{site.title}}">
+    <meta property="og:image" content="{{site.url}}/{{{uiRootPath}}}/img/datastax-docs-banner.png">
```

Comment:
@josuebran viewing the draft source, with the `{{uiRootPath}}` variable being used, the URL is rendering as `https://docs.datastax.com/en/astra/../_/img/datastax-docs-banner.png` because of this, the final URL that's resolved is  `https://docs.datastax.com/en/_/img/datastax-docs-banner.png` (effectively going up one level).

This might be why the Streaming site just used full URLs https://github.com/DataStaxDocs/datastax-streaming-docs-site/blob/main/supplemental-ui/partials/head-meta.hbs#L51
(@ddieruf can you confirm?)

Reply:
Use full URLs